typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -114,11 +142,28 @@ async function sendMessage(
     body: msg,
   };
   const value = await gasCalc.estimatePaymentForMessage(expected);
-  console.log(`Sending message from ${origin} to ${destination}`);
-  const receipt = await app.sendHelloWorld(origin, destination, msg, value);
-  console.log(JSON.stringify(receipt.events || receipt.logs));
+
+  log('Sending message', { origin, destination });
+
+  await new Promise<ethers.ContractReceipt[]>((resolve, reject) => {
+    setTimeout(
+      () => reject(new Error('Timeout waiting for message receipt')),
+      MESSAGE_RECEIPT_TIMEOUT,
+    );
+    app
+      .sendHelloWorld(origin, destination, msg, value, (receipt) => {
+        log('Message sent', {
+          origin,
+          destination,
+          events: receipt.events,
+          logs: receipt.logs,
+        });
+      })
+      .then(resolve)
+      .catch(reject);
+  });
 }
 
 main()
-  .then(() => console.info('HelloWorld sent'))
-  .catch(console.error);
+  .then(() => log('HelloWorld sent'))
+  .catch(error);
```

Comment:
bc of the aforementioned issue when stringifying errors, should probably do something similar to https://github.com/abacus-network/abacus-monorepo/pull/843/files#diff-10f7a1d27ccc4927804c7965831ff7b41986237dab3a841be1378911733293f3R251 here as well